common/app/views/fragments/head.scala.html

```diff
@@ -6,10 +10,21 @@
 @fragments.metaData(metaData)
 @head
 
-@fragments.javaScriptFirstSteps(metaData)
+<!--[if lt IE 9]>
+    <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
+<![endif]-->
 
-@fragments.stylesheetsLoad(projectName)
+@* get the stylesheets downloading ASAP *@
+@fragments.stylesheetLinks(projectName)
 
-@fragments.stylesheetsUse()
+@* polyfill, feature detect etc before we try and use the stylesheets *@
+@fragments.javaScriptPreFlight(metaData)
```

Comment:
Are we overloading the term preflight? Perhaps not… actually I think I quite like it :tongue: